{"schemaVersion":{"minor":3,"patch":0,"major":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"encapsulationKeyInconsistent"}],"platforms":["macOS"],"languages":["swift"]}]}],"identifier":{"url":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception\/encapsulationKeyInconsistent","interfaceLanguage":"swift"},"sections":[],"hierarchy":{"paths":[["doc:\/\/SwiftKyber\/documentation\/SwiftKyber","doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception"]]},"abstract":[{"type":"text","text":"Inconsistent encapsulation key data"}],"metadata":{"role":"symbol","symbolKind":"case","modules":[{"name":"SwiftKyber"}],"title":"Exception.encapsulationKeyInconsistent","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"encapsulationKeyInconsistent"}],"externalID":"s:10SwiftKyber9ExceptionO28encapsulationKeyInconsistentyA2CmF","roleHeading":"Case"},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swiftkyber\/exception\/encapsulationkeyinconsistent"]}],"kind":"symbol","references":{"doc://SwiftKyber/documentation/SwiftKyber/Exception/encapsulationKeyInconsistent":{"url":"\/documentation\/swiftkyber\/exception\/encapsulationkeyinconsistent","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception\/encapsulationKeyInconsistent","fragments":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"text":"encapsulationKeyInconsistent","kind":"identifier"}],"kind":"symbol","type":"topic","role":"symbol","title":"Exception.encapsulationKeyInconsistent","abstract":[{"type":"text","text":"Inconsistent encapsulation key data"}]},"doc://SwiftKyber/documentation/SwiftKyber":{"type":"topic","kind":"symbol","abstract":[{"type":"text","text":"Module-Lattice-Based Key-Encapsulation Mechanism Standard"}],"url":"\/documentation\/swiftkyber","title":"SwiftKyber","role":"collection","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber"},"doc://SwiftKyber/documentation/SwiftKyber/Exception":{"type":"topic","identifier":"doc:\/\/SwiftKyber\/documentation\/SwiftKyber\/Exception","navigatorTitle":[{"kind":"identifier","text":"Exception"}],"title":"Exception","role":"symbol","url":"\/documentation\/swiftkyber\/exception","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"text":"Exception","kind":"identifier"}],"kind":"symbol","abstract":[{"type":"text","text":"The Kyber exceptions"}]}}}